ui/src/main/java/com/dat3m/ui/options/OptionsPane.java

```diff
@@ -120,23 +137,32 @@ private void mkGrid(){
         cflagsPane.add(new JLabel("CFLAGS: "));
         cflagsPane.add(cflagsField);
 
+        JPanel configPane = new JPanel(new FlowLayout(LEFT));
+        configPane.add(new JLabel("Config:  "));
```

Comment:
I think "Config" is confusing. I would use "Options" which is how we refer to those in the README